tests/test_translation_plugin.py

```diff
@@ -1,170 +1,90 @@
-    assert len(instructions) == 1
-    assert c.find_bit(instructions[0].qubits[0]).index == 1
-    op = instructions[0].operation
-    assert isinstance(op, Reset)
-
-
-def test_int_1() -> None:
-    c = QuantumCircuit(2, 1)
-    c.initialize(1, 0)
-    new_c = _pm.run(c)
-    instructions = new_c.get_instructions('initialize')
-    assert len(instructions) == 1
-    assert c.find_bit(instructions[0].qubits[0]).index == 0
-    op = instructions[0].operation
-    assert isinstance(op, Initialize)
-    assert np.array_equal(op.params, [0, 1])
-
-
-def test_complex_plus() -> None:
-    c = QuantumCircuit(2, 1)
-    state = np.array([1, 1]) / np.sqrt(2)
-    c.initialize(state, 0)
-    new_c = _pm.run(c)
-    instructions = new_c.get_instructions('initialize')
-    assert len(instructions) == 1
-    assert c.find_bit(instructions[0].qubits[0]).index == 0
-    op = instructions[0].operation
-    assert isinstance(op, Initialize)
-    assert np.array_equal(op.params, state)
-
-
-def test_complex_plus_with_global_phase() -> None:
-    c = QuantumCircuit(2, 1)
-    state = np.array([1, 1]) / np.sqrt(2) * np.exp(3j)
-    c.initialize(state, 0)
-    new_c = _pm.run(c)
-    instructions = new_c.get_instructions('initialize')
-    assert len(instructions) == 1
-    assert c.find_bit(instructions[0].qubits[0]).index == 0
-    op = instructions[0].operation
-    assert isinstance(op, Initialize)
-    assert np.array_equal(op.params, state)
-
-
-def test_complex_minus() -> None:
-    c = QuantumCircuit(2, 1)
-    state = np.array([1, -1]) / np.sqrt(2)
-    c.initialize(state, 0)
-    new_c = _pm.run(c)
-    instructions = new_c.get_instructions('initialize')
-    assert len(instructions) == 1
-    assert c.find_bit(instructions[0].qubits[0]).index == 0
-    op = instructions[0].operation
-    assert isinstance(op, Initialize)
-    assert np.array_equal(op.params, state)
-
-
-def test_complex_0() -> None:
-    c = QuantumCircuit(2, 1)
-    c.initialize([1, 0], 0)
-    new_c = _pm.run(c)
-    instructions = new_c.get_instructions('reset')
-    assert len(instructions) == 1
-    assert c.find_bit(instructions[0].qubits[0]).index == 0
-    op = instructions[0].operation
-    assert isinstance(op, Reset)
-
-
-def test_complex_1() -> None:
-    c = QuantumCircuit(2, 1)
-    c.initialize([0, 1], 0)
-    new_c = _pm.run(c)
-    instructions = new_c.get_instructions('initialize')
-    assert len(instructions) == 1
-    assert c.find_bit(instructions[0].qubits[0]).index == 0
-    op = instructions[0].operation
-    assert isinstance(op, Initialize)
-    assert np.array_equal(op.params, [0, 1])
-
-
-def test_complex_unsupported() -> None:
-    c = QuantumCircuit(2, 1)
-    state = np.array([1, -3j])
-    state /= np.sqrt(np.sum(state * np.conj(state)))
-    c.initialize(state, 0)
-    with pytest.raises(AliceBobTranspilationException):
-        _pm.run(c)
+
+def _check_initialize(circuit: QuantumCircuit, expected: str) -> None:
```

Comment:
I'd say we could have a more explicit naming of this function and the one below.
How about `_assert_one_initialize` and `_assert_many_initializes`? (assert_one and assert_many seems to be the common structure in the Python test modules)

Reply:
Done!